### PR TITLE
mention peregrine in quickstart guide

### PR DESCRIPTION
**Pull request by weichweich**
## fixes https://github.com/KILTprotocol/ticket/issues/2674

* reword a few things using AI
* mention peregrine as a development chain

### CODE REVIEW

File: docs/develop/01_sdk/01_quickstart.md [hunk 1]
```diff
@@ -5,39 +5,40 @@ title: Quickstart
...
-First, we need to create a new project in a new directory.
-For this, we run `mkdir kilt-rocks && cd kilt-rocks`.
+Let's start by creating a new project from scratch.
+Create a new project in a fresh directory by running `mkdir kilt-rocks && cd kilt-rocks`.
 
 <Tabs groupId="ts-js-choice">
   <TabItem value='ts' label='Typescript' default>
 
-  From inside the `kilt-rocks` project directory, install the **KILT SDK**, **Ts-node**, **Axios** and **Typescript**:
+  From inside the `kilt-rocks` project directory, install the **KILT SDK**, **Typescript**, **ts-node** and **Axios**:
```
Comment (ntn-x2) [inline docs/develop/01_sdk/01_quickstart.md:41]: Should we then also change the order of the installation in the `npm install` command below?

File: docs/develop/01_sdk/01_quickstart.md [hunk 2]
```diff
@@ -133,34 +136,49 @@ We will move onto connecting to the **KILT blockchain**.
...
   className="language-ts"
-  funcEnd="return"
 >
-  {Connect}
+  {Disconnect}
 </SnippetBlock>
 
-After establishing a connection, you have access to the chain, but let's not forget to **close** any connections when we are done!
-Connections to blockchain nodes should be dropped when no longer needed: to do that simply call the `Kilt.disconnect()` function at the bottom of `main` function.
+By adding `await Kilt.disconnect()` at the end of the main function, you ensure that the connection to the blockchain node is properly closed when the script finishes executing.
```
Comment (ntn-x2) [inline docs/develop/01_sdk/01_quickstart.md:173]:
```suggestion
By adding `await Kilt.disconnect()` at the end of the main function, you ensure that the connection to the blockchain node is properly closed when the script terminates execution.
```

File: docs/develop/01_sdk/01_quickstart.md [hunk 3]
```diff
@@ -133,34 +136,49 @@ We will move onto connecting to the **KILT blockchain**.
...
+This helps maintain the integrity of your application and is a good practice to follow.
```
Comment (ntn-x2) [inline docs/develop/01_sdk/01_quickstart.md:174]: What does integrity means?

File: docs/develop/01_sdk/01_quickstart.md [hunk 4]
```diff
@@ -133,34 +136,49 @@ We will move onto connecting to the **KILT blockchain**.
...
-  className="language-ts"
->
-  {Disconnect}
-</SnippetBlock>
 
-**Congratulations!
-You have connected to a Spiritnet node.
-Let's now start querying some data from the chain!**
+Congratulations!
+You have connected to a blockchain node.
```
Comment (ntn-x2) [inline docs/develop/01_sdk/01_quickstart.md]: maybe?
```suggestion
You have connected to a KILT blockchain node.
```

**Review by ntn-x2 — APPROVED** [review]

File: docs/develop/01_sdk/01_quickstart.md (hunk 3 above)
Comment (weichweich) [reply docs/develop/01_sdk/01_quickstart.md:174]: keeping the app state free from resource leaks which could lead to a crash. [Integrity as in an unimpaired condition](https://www.merriam-webster.com/dictionary/integrity)

**Pull request merged**